db/migrate/20150304033052_add_bonus_to_grade_entry_items.rb

```diff
@@ -0,0 +1,5 @@
+class AddBonusToGradeEntryItems < ActiveRecord::Migration
+  def change
+  	add_column :grade_entry_items, :bonus, :boolean, :default => false
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 (not 1) spaces for indentation.<br>Tab detected.